RHDEVDOCS-5147:Doc; [ODC] Document column management for the TaskRuns and PipelineRuns list pages (#61267)
openshift/openshift-docs: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc

* On the *Pipeline Run details* page, click the *Task Runs* tab to see the completed, running, and failed runs for the task.
+
The *Task Runs* tab provides information about the task run along with the links to its task and pod, and also the status and duration of the task run. Use the Options menu {kebab} to delete a task run.
+
[NOTE]
====

> The *TaskRuns* list page features a *Manage columns* button, which you can also use to add a *Duration* column.

Reviewer: ```suggestion
The *Task Runs* list page features a *Manage columns* button, which you can also use to add a *Duration* column.
```

Author: Hi @max-cx! The UI shows the *TaskRuns* instead of *Task Runs*. Is it okay to still change it?

Reviewer: Of course, whatever is in the UI. The reason I made this suggestion was that the lines before and after in this file discuss a *Task Runs*, which is perhaps a different UI element. In any case, I wanted to bring this to your attention just to be safe.

> You can manage columns available on the *TaskRuns* list page using the *Manage columns* icon. By using the *Manage columns* option, you can add the *Duration* column which is not present by default, to the *TaskRuns* list page.

Author: Noted! Thank you for the suggestion. I'll make the changes! :)

> You can manage columns available on the *Pipeline Run details* and *TaskRuns* details pages using the *Manage columns* icon. By using the *Manage columns* option, you can add the *Duration* column that is not present by default, to the *Pipeline Run details* and *TaskRuns* details pages.

Reviewer 2: ```suggestion
You can manage columns available on the *TaskRuns* list page using the *Manage columns* icon. By using the *Manage columns* option, you can add the *Duration* column which is not present by default, to the *TaskRuns* list page.
```

Author: Made this change. Thanks!
